dotcom-rendering/src/server/server.dev.ts

```diff
@@ -95,7 +98,9 @@ export const devServer = (): Handler => {
 			case 'EditionsCrossword':
 				return handleEditionsCrossword(req, res, next);
 			case 'FootballDataPage':
-				return handleFootballDataPage(req, res, next);
+				return handleFootballMatchListPage(req, res, next);
+			case 'FootballTablesPage':
+				return handleFootballTablesPage(req, res, next);
```

Comment:
We may have to rename `FootballDataPage` to something like `FootballMatchListPage` if we've got a separate path for `FootballTablesPage`? This will need to be co-ordinated with frontend though, as it's already using this path.

Reply:
Since it needs a change in frontend, shall we leave this to a separate PR? I created an [issue](https://github.com/orgs/guardian/projects/88/views/15?sliceBy%5Bvalue%5D=Decommission+Frontend+Rendering&pane=issue&itemId=105507746&issue=guardian%7Cdotcom-rendering%7C13751) for it